internal/discover/ipc.go

```diff
@@ -34,10 +39,7 @@ func NewIPCDiscoverer(logger logger.Interface, driverRoot string) (Discover, err
 			lookup.WithCount(1),
 		),
 		driverRoot,
-		[]string{
-			"/nvidia-persistenced/socket",
-			"/nvidia-fabricmanager/socket",
```

Comment:
Was removal of the fabricmanager socket intended?

Reply:
No. Updated.